frontend/src/data_context_global.tsx

```diff
@@ -74,6 +74,14 @@ export interface Herd {
     individuals?: Individual[]
 }
 
+export function individualLabel(individual: Individual | LimitedIndividual): string {
```

Comment:
`individual: LimitedIndividual` is enough here, you can still call it with an `Individual`

Reply:
good point!